src/picongpu/include/plugins/radiation/Radiation.kernel

```diff
@@ -234,145 +238,143 @@ void kernelRadiationParticles(ParBox pb,
                  * LATER: can this be optimized?
                  */
                 int saveParticleAt = -1;
-                PMACC_AUTO(par,(*frame)[linearThreadIdx]);
 
-                /* if radiation is not calculated for all particles
-                 * but not via the gamma filter, check which particles
-                 * have to be used for radiation calculation
-                 */
+                /* if particle is not accelerated we skip all calculations */
+                if( particle_momentumNow != particle_momentumOld )
```

Comment:
Thats a true point. And you are right, by definition two vectors are *unequal* if *any component* is *not equal*. Thus, `!=` follows the mathematical standard here.